src/OrbitVersion/OrbitVersion.cpp.in

```diff
@@ -14,9 +14,12 @@
   "@BUILD_OS_NAME@ @BUILD_OS_RELEASE@ @BUILD_OS_VERSION@ " \
   "(@BUILD_OS_PLATFORM@)"
 #define OrbitCommitHashStr "@COMMIT_HASH@"
+#define OrbitMajorVersion @MAJOR_VERSION@
+#define OrbitMinorVersion @MINOR_VERSION@
 
 namespace orbit_version {
 
+Version GetVersionNumber() { return {OrbitMajorVersion, OrbitMinorVersion}; }
 std::string GetVersion() { return OrbitVersionStr; }
```

Comment:
I think the naming could have been improved: `GetVersion` doesn't return a `Version`, but a `string`, while `GetVersionNumber` doesn't return a number, but a `Version`.

I think it would be
`GetVersion` -> `Version`
`GetVersionString` -> `std::string`

Reply:
will fix in the followup PR.